saraho1123/currency-exchange
# Fix gh-pages and npm start problem

## Is this a fix or a feature?
fix

## What does it fix?
Fixed a problem with deploying with gh-pages. It somehow also broke the rendering using `npm start`. I pretty much just re-did the set up for gh-pages and everything is working now.

## Where should the reviewer start?
+ line 5 of package.json
+ 94 of App.js (remove exact)

## How should this be tested?
+ clone down
+ cd into project
+ run `npm install` in terminal
+ run `npm start` in terminal
  - Form should appear. It can be filled out and button clicked, and user taken to `/currency-cards`.
  - All other functionality should still be in place.

## Relevant Tickets
n/a
